metricbeat/beater/metricbeat.go

```diff
@@ -1,8 +1,11 @@
 /*
+Package beater provides the implementation of the libbeat Beater interface for
```

Comment:
Godoc/golint is quite strict. There can only be one package level comment and it must begin with`Package <packagename>`.

- https://github.com/golang/go/wiki/CodeReviewComments#package-comments
- https://github.com/golang/lint/blob/master/lint.go#L418